sdk/metric/doc.go

```diff
@@ -39,6 +39,30 @@
 // Meter.RegisterCallback and Registration.Unregister to add and remove
 // callbacks without leaking memory.
 //
+// # Cardinality Limits
+//
+// Cardinality refers to the number of unique attribute combinations for a single
+// metric within one collection (export) interval. High cardinality can lead to
```

Comment:
This is only the case for delta temporality.

```suggestion
// Cardinality refers to the number of unique attributes collected. High cardinality can lead to
```